tests/integration.rs

```diff
@@ -2429,7 +2429,7 @@ mod common_parallel {
         assert!(
             String::from_utf8_lossy(&host_cpus_count.stdout)
                 .trim()
```

Comment:
The commit message subject should start with `tests:` it should be wrapped at 50.

The body should be wrapped at 72 (long URLs or copy and pasted error messages are the exceptions)

A better commit would like something like.

```
tests: Correctly parse number of host cpus > 255

test_cpu_affinity needs the number of host CPUs. Since it is possible
for the host to have more than 255 CPUs; increase the size of the
datatype used for parsing the string to accomodate this.
```

Reply:
I've updated the commit message and documented the rules.